docs/docs/releases/0.2.0-testing.1.md

```diff
@@ -0,0 +1,234 @@
+# 🧪 Mathesar 0.2.0-testing.1
+
+!!! danger "Not a stable release"
+    This is a testing build released with the goal of gathering feedback from our community. It has **[many known issues](https://github.com/mathesar-foundation/mathesar/issues?q=is%3Aissue+milestone%3Av0.2.0-testing.2+)** and is not recommended for production use.
```

Comment:
I already added a different admonition to the homepage that says this is a testing build, I think it would look weird to have two separate admonitions. If you do think we should add it, could you point out where it should go?

Reply:
On the installation pages -> Docker compose and install from scratch pages.

Reply:
Makes sense, added them there.